pkg/oci/load_balancer_util.go

```diff
@@ -236,3 +238,24 @@ func parseSecretString(secretString string) (string, string) {
 	}
 	return "", secretString
 }
+
+func getLoadBalancerSourceRanges(config *client.Config, service *api.Service) ([]string, error) {
+	sourceRanges, err := apiservice.GetLoadBalancerSourceRanges(service)
+	if err != nil {
+		return []string{}, err
+	}
+
+	// NOTE: This means that if DefaultSourceCIDRs are set a user cannot set
+	// a source CIDR range of allow all (i.e. 0.0.0.0/0).
+	if len(sourceRanges) == 1 && apiservice.IsAllowAll(sourceRanges) &&
```

Comment:
This means you can't set it to `0.0.0.0/0`. If your default is less secure, but you want this to be public it'd never work.